src/actions/users.js

```diff
@@ -675,6 +675,31 @@ export function revokeSession(userId, sessionId) {
     };
 }
 
+export function revokeAllSessionsForUser(userId) {
+    return async (dispatch, getState) => {
+        dispatch({type: UserTypes.REVOKE_ALL_USER_SESSIONS_REQUEST}, getState);
+
+        try {
+            await Client4.revokeAllSessionsForUser(userId);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch);
+            dispatch(batchActions([
+                {type: UserTypes.REVOKE_ALL_USER_SESSIONS_FAILURE, error},
+                logError(error)(dispatch)
+            ]), getState);
+            return false;
```

Comment:
We're actually trying to use new pattern for action returns (#265 is actually switching them all over to it), which will allow easier access to errors. So can I ask you to do `return {error};` here and instead of returning true below, do `return {data: true};`